### PR TITLE
Minor stuff

### PR DESCRIPTION
**Pull request by tom-rosewell**

### CODE REVIEW

File: ValeStyles/Couchbase/An.yml [hunk 1]
```diff
@@ -14,4 +14,6 @@ exceptions:
   - '[Aa] [Uu]ser'
   - '[Aa] [Uu]sername'
   - '[Aa] UI'
-  - '[Aa] <<[BbCcDdFfGgHhJjKkLlMmNnPpQqRrSsTtVvWwXxYyZz].+>>'
\ No newline at end of file
+  - '[Aa] <<[BbCcDdFfGgHhJjKkLlMmNnPpQqRrSsTtVvWwXxYyZz].+>>'
+  - '[Aa] [Oo]ne'
+  - '[Aa] [Oo]nce'
```
Comment (sarahlwelton) [inline ValeStyles/Couchbase/An.yml:19]: Curious why you've added this as an exception? I'm not seeing how it's grammatically correct right now.

Comment (tom-rosewell) [reply ValeStyles/Couchbase/An.yml:19]: That one's not from an occurrence, I was just adding "one" and thought I should add that too. Is it not grammatically correct?

Comment (tom-rosewell) [reply ValeStyles/Couchbase/An.yml:19]: It's probably not going to affect much if it's removed, something like "a once underutilised feature" is the only phrase I could see it being used in.

Comment (sarahlwelton) [reply ValeStyles/Couchbase/An.yml:19]: Hmm, brain just wasn't quite thinking of a sentence that made sense. Thanks for the example.

**Review by sarahlwelton — APPROVED** [review]

**Pull request merged**